src/containers/PaymentInfo/PaymentInfo.component.js

```diff
@@ -164,15 +165,15 @@ const PaymentInfoFn = ({
 
     return () => {
       hideInnerPopup();
+      // replace with: hide payment info popup?
```

Comment:
I think that yes, as the payment info popup is the only popup in the PaymentInfo component